<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.0000" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2024-09-05T20:55:32" tests="0" file="cypress/integration/plugins/index-management-dashboards-plugin/rollups_spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Rollups" timestamp="2024-09-05T20:55:32" tests="0" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="can be created" timestamp="2024-09-05T20:55:32" tests="1" time="0.0000" failures="1">
    <testcase name="Rollups can be created &quot;before each&quot; hook for &quot;successfully&quot;" time="0.0000" classname="&quot;before each&quot; hook for &quot;successfully&quot;">
      <failure message="Timed out retrying after 60000ms: Expected to find content: &apos;Sample data&apos; but never did.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Rollups`" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 60000ms: Expected to find content: 'Sample data' but never did.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Rollups`
    at Context.eval (http://localhost:5601/__cypress/tests?p=cypress/integration/plugins/index-management-dashboards-plugin/rollups_spec.js:184:8)]]></failure>
    </testcase>
  </testsuite>
</testsuites>